include/picongpu/particles/ionization/byField/IonizationCurrent/JIonizationCalc.hpp

```diff
@@ -0,0 +1,40 @@
+/* Copyright 2020 Jakob Trojok
```

Comment:
Same as before: EOF blank line, end of namespace comments, doxygen documentation of struct and `operator()`